google-cloud-datastore/src/test/java/com/google/cloud/datastore/it/ITE2ETracingTest.java

```diff
@@ -319,6 +337,7 @@ public void after() throws Exception {
     if (isUsingGlobalOpenTelemetrySDK()) {
       GlobalOpenTelemetry.resetForTest();
     }
+    remoteDatastoreHelper.deleteNamespace();
```

Comment:
I think this should take place in `@AfterClass` rather than `@After` (unless there is a different namespace per test? But ideally there should be one namespace for the entire suite)

Reply:
Actually this helps reuse of a fixed set of keys in every test case. Are namespaces expensive?

Reply:
Your thoughts on the above comment @kolea2 ?

Reply:
sorry for missing this  - I was thinking we should align with ITDatastoreTest, but if you find it's better to leave this as is for these test cases, that works for me.